chat-widget/src/components/livechatwidget/common/initWebChatComposer.ts

```diff
@@ -61,8 +62,13 @@ export const initWebChatComposer = (props: ILiveChatWidgetProps, chatSDK: any, s
             if (props?.webChatContainerProps?.renderingMiddlewareProps?.hideSendboxOnConversationEnd !== false) {
                 setWebChatStyles((styles: StyleOptions) => { return { ...styles, hideSendBox: true }; });
             }
+            WebChatStoreLoader.store = null;
```

Comment:
Why do we need this line?

Reply:
In case of agent ends conversation, we don't call `endChat` or `prepareEndChat`. This leaves `WebChatStoreLoader.store` un-empty before moving to post chat state. This also creates a prob that post chat loading is called repeatedly. Some sort of polling happening in background I believe.